packages/tools/client-debugger/client-debugger/src/messaging/TelemetryMessages.ts

```diff
@@ -8,6 +8,22 @@ import { IDebuggerMessage } from "./Messages";
 
 // #region Outbound messages
 
+/**
+ * Interface for telemetry events with a timestamp. Specific to the Fluid Debugger.
+ * @public
+ */
+export interface ITimestampedTelemetryEvent {
+	/**
+	 * The contents of the telemetry event.
+	 */
+	logContent: ITelemetryBaseEvent;
+
+	/**
+	 * The timestamp of the telemetry event.
```

Comment:
Can we document the expected format for this time stamp?

Reply:
We can document it in terms of `Date.now` if that's how we're populating it.

Reply:
Added the example format and {@link Date.now()}